cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java

```diff
@@ -288,7 +290,7 @@ public void consume(Context context, Purchase purchase, ConsumeListener listener
   }
 
   @Override
-  public void getInventory(Context context, Collection<String> itemSkus, Collection<String> subSkus,
+  public void getInventory(Context context, Collection<String> inappSkus, Collection<String> subSkus,
```

Comment:
What is the reason for this change from `itemSkus` to `inappSkus` across this file?

Reply:
Just noticed that 
`
  public static final String PRODUCT_TYPE_ITEM = "inapp";
  public static final String PRODUCT_TYPE_SUBSCRIPTION = "subs";
`
So `itemSkus -> inappSkus` felt reasonable. Can revert back, or `PRODUCT_TYPE_ITEM -> PRODUCT_TYPE_INAPP` for consistency